docker/dockerfile

```diff
@@ -7,6 +9,14 @@ RUN --mount=target=/requirements,type=bind,source=requirements <<EOF
 
     apt update
 
+    # Be sure to install a recent CMake, see also https://apt.kitware.com/.
```

Comment:
Use something like
```docker
If not $(apt --yes --no-install-recommends satisfy "cmake (>=3.23)" then
   Add kitware
   Apt install cmake
Endif
```

Reply:
Or simply get the binaries at https://cmake.org/download/, it's gonna be easier.

Reply:
OK, done.